### PR TITLE
Add parallax stories

### PR DESCRIPTION
**Pull request by onluiz**
# Description

Add stories for component Parallax.

## Type of change

# How Has This Been Tested?

- Unit testing
- Browser testing

# Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have not generated a new package version. (the maintainers will handle that)

### CODE REVIEW

File: stories/Parallax.stories.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Parallax from '../src/Parallax';
+
+const stories = storiesOf('javascript/Parallax', module);
```
Comment (alextrastero) [inline stories/Parallax.stories.js]: Can this be `const stories = storiesOf('Javascript|Parallax', module);`

Comment (onluiz) [reply stories/Parallax.stories.js]: Fixed!

@alextrastero

Comment (alextrastero) [reply stories/Parallax.stories.js]: Has to be `Javascript|Parallax` sorry :( try to run it locally, you'll see how nice and organize it looks

Comment (onluiz) [reply stories/Parallax.stories.js]: Sorry :smile: I it will fix ASAP

@alextrastero

Comment (onluiz) [reply stories/Parallax.stories.js]: Fixed!

Comment (alextrastero) [reply stories/Parallax.stories.js]: u da man! thanks

**Pull request merged**